docs/howto/operate/new-tf-cluster.md

```diff
@@ -41,26 +41,18 @@ Then you can change into the terraform directory and initialise
 
 ```bash
 cd terraform
-terraform init
+terraform init -backend-config=backends/default-backend.hcl
 ```
 
 ````{note}
-If you are deploying a cluster to a project you have access to via a different account, such as a university-affliated account, there are a few extra steps to take.
-Hopefully, this will be a rare scenario.
+If you are working on a project which you cannot access with your 2i2c account, there are other backend config files stored in `terraform/backends` that will configure a different storage bucket to read/write the remote terraform state.
+This saves us the pain of having to handle multiple authentications as these storage buckets are within the project we are trying to deploy to.
 
-First, you will need to provide terraform with an [access token](https://www.terraform.io/docs/language/settings/backends/gcs.html#configuration-variables) to use the state files.
-You can generate one using the below commands and logging in with your 2i2c.org account.
+For example, to work with Pangeo you would initialise terraform like so:
 
 ```bash
-gcloud auth application-default login
-gcloud auth application-default print-access-token
+terraform init -backend-config=pangeo-backend.hcl
 ```
-
-Add the access token to the [terraform backend block](https://github.com/2i2c-org/pilot-hubs/blob/2ef8a4bf35bb5ee9bf04ab3db1218b8c183c5da2/terraform/main.tf#L2-L5) in `main.tf`.
-**DO NOT COMMIT THIS CHANGE.**
-Then run `terraform init` or `terraform init -reconfigure`.
-
-You can now login to your other gcloud account and proceed with the guide.
 ````
 
 ## Creating a new terraform workspace
```

Comment:
Let's add a note that workspaces are per backend? So if you don't see a particular workspace, it might be because you aren't in the right backend.

Is there a way to dynamically switch between backends on the same checkout? Or do we need different git clones?

Reply:
> Is there a way to dynamically switch between backends on the same checkout? Or do we need different git clones?

Backends doesn't have anything to do with git, you choose it when you run `terraform init -backend-config=path/to/config`

Reply:
> Let's add a note that workspaces are per backend? So if you don't see a particular workspace, it might be because you aren't in the right backend.

Done!

Reply:
@sgibson91 if I was working on the pilot hubs infra (in workspace `pilot-hubs` repo), and want to work on justiceinnovation lab instead (in workspace `justiceinnovationlab`), I can run `terraform workspace switch justiceinnovationlab`. But that only works if they're in the same backend. If they're in a different backend, how would this switch happen?

Reply:
```
terraform init -backend-config=/path/to/backend/config/that/contains/that/workspace
terraform workspace select desiredworkspace
```

At the minute we only have two backends: our default one that contains all our currently known workspaces, and the Pangeo one on the Columbia project, that only contains one workspace for the Pangeo hubs.

Reply:
I was having to re-run `terraform init` _a lot_, because every time I needed to update my `access_token` terraform would recognise this as a change in the backend and ask to be reinitialised. [This comment](https://github.com/hashicorp/terraform/issues/17288#issuecomment-363851018) might explain it a bit better.

Reply:
This is why I originally didn't abstract out our default backend, because Pangeo is a special case.

Reply:
We may also want to agree on some team process around whether we prefer `-reconfigure` or `-migrate-state` when rerunning `terraform init`. https://www.terraform.io/docs/cli/commands/init.html#backend-initialization

Reply:
terraform is also smart enough to know which workspaces exist where

```
tf init -backend-config=backends/pangeo-backend.hcl -reconfigure

Initializing the backend...

Successfully configured the backend "gcs"! Terraform will automatically
use this backend unless the backend configuration changes.

The currently selected workspace (sandbox) does not exist.
  This is expected behavior when the selected workspace did not have an
  existing non-empty state. Please enter a number to select a workspace:
  
  1. default
  2. pangeo-hubs

  Enter a value:
```

Reply:
Reading it, I think we should suggest doing `terraform init -reconfigure`, since we'll just be switching workspaces. Let's document that as 'how to switch workspaces', and then we can merge this?

Thank you so much for working on this! It'll save a lot of pain going forward :D

Reply:
> Let's document that as 'how to switch workspaces', and then we can merge this?

Done! Will merge tomorrow if I receive no further comments. I've linked to this in the team-sync issue and I want to make sure the team know this is happening since it changes our workflow.